pkg/windows/calico.go

```diff
@@ -59,26 +61,31 @@ func (c *Calico) Setup(ctx context.Context, dataDir string, nodeConfig *daemonco
 	if err != nil {
 		return nil, err
 	}
-
-	secrets, err := client.CoreV1().Secrets(CalicoSystemNamespace).List(ctx, metav1.ListOptions{})
+	serviceAccounts := client.CoreV1().ServiceAccounts(CalicoSystemNamespace)
+	calicoSA, err := serviceAccounts.Get(ctx, calicoNode, metav1.GetOptions{})
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "failed to create token for service account %s/%s", calicoSA.Namespace, calicoSA.Name)
+	}
+
+	req := authv1.TokenRequest{
+		Spec: authv1.TokenRequestSpec{
+			BoundObjectRef: &authv1.BoundObjectReference{
+				Kind:       calicoSA.Kind,
+				APIVersion: calicoSA.APIVersion,
+				Name:       calicoSA.Name,
+			},
+			Audiences:         []string{version.Program},
+			ExpirationSeconds: pointer.Int64(3600 * 365),
```

Comment:
I think this is 365 hours instead of 365 days
```suggestion
			ExpirationSeconds: pointer.Int64(60 * 60 * 24 * 365),
```

Reply:
oops, I can't do math. fixing